tests/test_predictit.py

```diff
@@ -0,0 +1,93 @@
+import datetime
+
+import pandas as pd
+
+import ergo
+
+
+def test_refresh(predictit):
```

Comment:
The problem with trying to test refreshing is that calling refresh doesn't actually guarantee any changes because the api is only updated once every minute. My reasoning behind the test was to make sure the program doesn't fetch a different market or question upon calling refresh by messing with the id.

Reply:
cool -- I'd say:
1. Is there anything that *is* guaranteed to change on refresh? If so, test that
2. If not, just delete this test